src/kernel/log.zig

```diff
@@ -0,0 +1,37 @@
+const serial = @import("serial.zig");
+const fmt = @import("std").fmt;
+
+pub const Level = enum {
+    INFO,
+    DEBUG,
+    WARNING,
+    ERROR
+};
+
+pub fn init() void {
+    serial.init(serial.DEFAULT_BAUDRATE, serial.Port.COM1) catch unreachable;
```

Comment:
Its probs best to init the serial separately as other stuff may use the serial without the logging

Reply:
Very true